js/interactive-guides/microprofile-config/microprofile-config-callback.js

```diff
@@ -10,11 +10,11 @@
 *******************************************************************************/
 var microprofileConfigCallBack = (function() {
 
-    var propsFileConfig = "download-url=ftp://music.com/canada/download";
+    var propsFileConfig = "download_url=ftp://music.com/canada/download";
```

Comment:
You'll have merge problem as you're changing all the download-url to download_url in the file. I understand why you're doing it but the changes are already delivered in the dev branch with the correct download_url property name and the ftp urls were corrected too as canada and asia are not used anymore. You're behind with the dev branch. I though you'll pull and rebase before making your changes. Look like you did not rebase first. With that, you should only touch the download-url for your changes only, not to change the entire file. When you resolve the merge conflict, be careful and pick the right one. Then make sure to test all the affected steps.